lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -693,6 +1102,10 @@ pub enum CGOPointerWheel {
 
 impl From<CGOMousePointerEvent> for PointerEvent {
     fn from(p: CGOMousePointerEvent) -> PointerEvent {
+        // # Safety
```

Comment:
Comments like this generally belong on the function, and using `///`, not placed in the body of the function.